package.json

```diff
@@ -74,7 +74,10 @@
         "mini-css-extract-plugin": "^0.8.0",
         "mock-local-storage": "^1.1.15",
         "mojito-rb-gen": "^0.0.1",
-        "node-sass": "4.13.1",
+        "sass": "1.34.1",
```

Comment:
@bfoxx1906, I'd suggest doing this in a separate PR. Did we validate the CSS output is identical between the old and new libraries?

Reply:
This is what what we're using in buie right now but that's a good point. I'll remove that and then verify the css.